grants/search.js

```diff
@@ -264,19 +285,53 @@ function buildLocationFacet(geoCodeType) {
  * Build $facet criteria
  * @see https://docs.mongodb.com/manual/reference/operator/aggregation/facet/index.html
  */
+
+const amountAwardedBuckets = [0, 10000, 50000, 100000, 1000000, Infinity];
+
 function buildFacetCriteria() {
     return {
-        localAuthorities: buildLocationFacet('CMLAD'),
-        westminsterConstituencies: buildLocationFacet('WPC'),
+
+        countries: [
+            {
+                $project: {
+                    items: {
+                        $filter: {
+                            input: '$beneficiaryLocation',
+                            as: 'location',
+                            cond: { $eq: ['$$location.geoCodeType', 'CMLAD'] }
```

Comment:
It might be worth defining CMLAD and WPC as constants somewhere so we can name them, we use them a couple times.